PROJECT

```diff
@@ -12,16 +12,19 @@ resources:
   domain: keda.sh
   group: keda
   kind: ScaledObject
-  path: github.com/kedacore/keda/v2/apis/keda/v1alpha1
+  path: github.com/kedacore/keda/apis/keda/v1alpha1
```

Comment:
The problem was that operator-framework/kubebuilder didn't detect the APIs and i couldn't create the webhook. Once I changed this, the tooling worked again, but honestly, I didn't understand way and maybe it was because I was doing something wrong, if you think that v2 is worth, I can revert this change

Reply:
ah okay, so keep it this way then!